frontend/public/components/api-explorer.tsx

```diff
@@ -578,35 +601,37 @@ const APIResourceAccessReview: React.FC<APIResourceTabProps> = ({
           onSelectAll={onSelectAll}
         >
           <CheckBox
-            title="User"
+            title={t('api-explorer~User')}
             active={showUsers}
             number={users.length}
             toggle={toggleShowUsers}
           />
           <CheckBox
-            title="Group"
+            title={t('api-explorer~Group')}
             active={showGroups}
             number={groups.length}
             toggle={toggleShowGroups}
           />
           <CheckBox
-            title="ServiceAccount"
+            title={t('api-explorer~ServiceAccount')}
             active={showServiceAccounts}
             number={serviceAccounts.length}
             toggle={toggleShowServiceAccounts}
           />
         </CheckBoxControls>
         <p className="co-m-pane__explanation">
-          The following subjects can {verb} {plural}
-          {namespaced && namespace && <> in namespace {namespace}</>}
-          {namespaced && !namespace && <> in all namespaces</>}
-          {!namespaced && <> at the cluster scope</>}.
+          {t('api-explorer~The following subjects can {{verb}} {{plural}}', { verb, plural })}
+          {namespaced && namespace && (
+            <> {t('api-explorer~in namespace {{ namespace }}', { namespace })}</>
+          )}
+          {namespaced && !namespace && <> {t('api-explorer~in all namespaces')}</>}
+          {!namespaced && <> {t('api-explorer~at the cluster scope')}</>}.
```

Comment:
The entire sentence needs to be one message with variables. (I realize we might need a couple different messages here depending on whether it's namespaced.)

Reply:
ok, @spadgett created 3 full messages based on logical differences.